editor.planx.uk/src/pages/FlowEditor/lib/store/shared.ts

```diff
@@ -55,8 +69,9 @@ export const sharedStore = (
     });
   },
 
-  setFlow(id, flow) {
-    set({ id, flow });
+  setFlow({ id, flow, flowSlug }) {
+    set({ id, flow, flowSlug });
+    set({ flowName: flowSlug.replaceAll?.("-", " ") });
```

Comment:
nit: I think this use of `replaceAll` to format the slug as `flowName` will return slightly different casing than `convertSlugToName` in the API saveAndReturn/utils - would be good to keep consistency!